lib/vdsm/storage/storageServer.py

```diff
@@ -935,11 +935,13 @@ def disconnect(dom_type, con_defs):
 
 def _prepare_connections(dom_type, con_defs):
```

Comment:
The assumption that the connection list is not empty is correct. You can
add an asset here to make this clear. Once we fix hsm.py to validate 
input, we will never get here empty list.

Reply:
we should fix the code not to potentially use undefined variable anyway - IDEs (like pycharm) warns about it and it can be confusing